cdap-data-fabric/src/main/resources/cdap/grok/patterns/firewalls

```diff
@@ -0,0 +1,60 @@
+# NetScreen firewall logs
```

Comment:
Wasn't sure how to handle the license here, as logstash stored these patterns under a directory that had the LICENSE in it, which is what I replicated under `cdap-data-fabric/src/main/resources/cdap/grok`. Having license headers here makes more sense though.